resources/config.yml

```diff
@@ -0,0 +1,4 @@
+---
+enabled-worlds: []
+
+disabled-worlds: []
```

Comment:
Needs a comment to let users know to put the folder name instead of the level name. It would also be a good idea to mention that it will be case sensitive (unless you are going to adjust for that in the code).